modin/numpy/test/test_array.py

```diff
@@ -145,6 +145,57 @@ def test_update_inplace():
     numpy.testing.assert_array_equal(out._to_numpy(), arr2._to_numpy())
 
 
+@pytest.mark.parametrize(
+    "data_out",
+    [
+        numpy.zeros((1, 3)),
+        numpy.zeros((2, 3)),
+    ],
+)
+def test_out_broadcast(data_out):
+    if data_out.shape == (2, 3):
+        pytest.xfail("broadcasting would require duplicating row: see GH#5819")
+    data1 = [[1, 2, 3]]
+    data2 = [7, 8, 9]
+    numpy_out = numpy.array(data_out)
+    numpy.add(numpy.array(data1), numpy.array(data2), out=numpy_out)
+    modin_out = np.array(data_out)
+    np.add(np.array(data1), np.array(data2), out=modin_out)
+    numpy.testing.assert_array_equal(modin_out._to_numpy(), numpy_out)
```

Comment:
Maybe it makes sense to create our own utility to compare arrays? (like `df_equals`)

if yes, could you create a separate issue for that?

Reply:
#5933

Reply:
thanks!